juju/charmstore
# Adding Version url with version number and git sha

QA:
make server, go to http://localhost:8080/v4/debug/info
Check that version and id are unknown and that you get a log saying that the version file is not found

Add a version.yaml in cm/charmd/ with version: something and id: something 
Check again it is displayed and no log are appearing.
